src/resmom/mom_main.c

```diff
@@ -10529,36 +10511,12 @@ active_idle(job *pjob, int which)
 		pjob->ji_qs.ji_svrflags |= JOB_SVFLG_Actsuspd;
 		send_wk_job_idle(pjob->ji_qs.ji_jobid, which);
 		if ((pjob->ji_qs.ji_svrflags &
-			(JOB_SVFLG_Suspend|JOB_SVFLG_Suspend)) == 0) {
-			/* save stop time for adjusting walltime */
-			pjob->ji_momstat = time_now;
```

Comment:
This pjob->ji_momstat is looking like not relevant anymore. Perhaps remove the field from the job structure?

Reply:
Ok. I wasn't sure if it's used for other purposes. I will check and remove it.

Reply:
Removed `ji_momstat`